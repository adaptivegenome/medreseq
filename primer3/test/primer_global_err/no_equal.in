SEQUENCE_ID=without_eq
AGTCGATCGTAGCTAGCTAGCTAGCTGACTGATGCTAGCTAGCTGATCGATCGTAGCTAGCTAGCTGATCGTAGCTAGCTAGCTGACTGATCATCTAGCTAGCTGATCATCGTAGCTAGCTCTATTTTATCGTAGCTAGCTATCATCTAGCTAGC
=
