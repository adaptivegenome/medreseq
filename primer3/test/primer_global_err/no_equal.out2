../src/primer3_core: Input line with no '=': AGTCGATCGTAGCTAGCTAGCTAGCTGACTGATGCTAGCTAGCTGATCGATCGTAGCTAGCTAGCTGATCGTAGCTAGCTAGCTGACTGATCATCTAGCTAGCTGATCATCGTAGCTAGCTCTATTTTATCGTAGCTAGCTATCATCTAGCTAGC
