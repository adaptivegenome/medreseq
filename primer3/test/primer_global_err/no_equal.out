SEQUENCE_ID=without_eq
AGTCGATCGTAGCTAGCTAGCTAGCTGACTGATGCTAGCTAGCTGATCGATCGTAGCTAGCTAGCTGATCGTAGCTAGCTAGCTGACTGATCATCTAGCTAGCTGATCATCGTAGCTAGCTCTATTTTATCGTAGCTAGCTATCATCTAGCTAGC
PRIMER_ERROR=Input line with no '=': AGTCGATCGTAGCTAGCTAGCTAGCTGACTGATGCTAGCTAGCTGATCGATCGTAGCTAGCTAGCTGATCGTAGCTAGCTAGCTGACTGATCATCTAGCTAGCTGATCATCGTAGCTAGCTCTATTTTATCGTAGCTAGCTATCATCTAGCTAGC
=
